concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-751	751	will nicht	want not	V. Adverbien	C. Der Art und Weise		
+Gabelentz-1892-1074-752	752	kann nicht	can not	V. Adverbien	C. Der Art und Weise		
+Gabelentz-1892-1074-753	753	darf nicht	may not	V. Adverbien	C. Der Art und Weise		
+Gabelentz-1892-1074-754	754	Kaum	hardly	V. Adverbien	C. Der Art und Weise	2926	HARDLY
+Gabelentz-1892-1074-755	755	Vergeblich	futile, in vain	V. Adverbien	C. Der Art und Weise	1765	IN VAIN
+Gabelentz-1892-1074-756	756	Zusammen	together	V. Adverbien	C. Der Art und Weise	2482	TOGETHER
+Gabelentz-1892-1074-757	757	Wieder	again	V. Adverbien	C. Der Art und Weise	204	AGAIN
+Gabelentz-1892-1074-758	758	Noch	still	V. Adverbien	C. Der Art und Weise	2919	STILL (CONTINUING)
+Gabelentz-1892-1074-759	759	Nur	only	V. Adverbien	C. Der Art und Weise	1580	ONLY
+Gabelentz-1892-1074-760	760	Und	and	VI. Conjunctionen		1577	AND
+Gabelentz-1892-1074-761	761	Oder	or	VI. Conjunctionen		1014	OR
+Gabelentz-1892-1074-762	762	Aber	but	VI. Conjunctionen			
+Gabelentz-1892-1074-763	763	Denn, weil	because	VI. Conjunctionen		1157	BECAUSE
+Gabelentz-1892-1074-764	764	Als	when	VI. Conjunctionen		1238	WHEN
+Gabelentz-1892-1074-765	765	Während, so lange als	while	VI. Conjunctionen		726	WHILE
+Gabelentz-1892-1074-766	766	Bis dass	until	VI. Conjunctionen		2753	UNTIL
+Gabelentz-1892-1074-767	767	Damit, um zu	in order to	VI. Conjunctionen			
+Gabelentz-1892-1074-768	768	Dass (Objektssatz)	that, which	VI. Conjunctionen			
+Gabelentz-1892-1074-769	769	Wie	how	VI. Conjunctionen		1239	HOW
+Gabelentz-1892-1074-770	770	Wenn	when	VI. Conjunctionen		1238	WHEN
+Gabelentz-1892-1074-771	771	Ob	if	VI. Conjunctionen		1459	IF
+Gabelentz-1892-1074-772	772	Obschon	although	VI. Conjunctionen			
+Gabelentz-1892-1074-773	773	Zu (Richtung wohin)	to (direction)	VII. Präpositionen		2754	TOWARDS
+Gabelentz-1892-1074-774	774	Dativ	(realization of dative case)	VII. Präpositionen			
+Gabelentz-1892-1074-775	775	Mit (in Begleitung von…)	with	VII. Präpositionen		1340	WITH
+Gabelentz-1892-1074-776	776	vermittels	by means of	VII. Präpositionen			
+Gabelentz-1892-1074-777	777	Durch	through	VII. Präpositionen		97	THROUGH
+Gabelentz-1892-1074-778	778	Gegen	against	VII. Präpositionen		2469	AGAINST
+Gabelentz-1892-1074-779	779	Wegen	because	VII. Präpositionen		1157	BECAUSE
+Gabelentz-1892-1074-780	780	Um (etwas herum)	around something	VII. Präpositionen			
+Gabelentz-1892-1074-781	781	Von (etwas her)	from (somewhere)	VII. Präpositionen		693	FROM
+Gabelentz-1892-1074-782	782	Genitiv	(realization of genitive case)	VII. Präpositionen			
+Gabelentz-1892-1074-783	783	In (etwas drinnen)	inside	VII. Präpositionen		1606	INSIDE
+Gabelentz-1892-1074-784	784	in (etwas hinein)	inside (movement)	VII. Präpositionen			
+Gabelentz-1892-1074-785	785	Für	for	VII. Präpositionen			
+Gabelentz-1892-1074-786	786	Bei (dicht bei, nahe bei)	nearby	VII. Präpositionen			
+Gabelentz-1892-1074-787	787	An (der Seite von…)	at (someone’s side)	VII. Präpositionen		1461	AT
```

Comment:
I'd rather map the Bei to AT.

Reply:
"an der Seite von" seems more specifically NEXT TO than AT.

Reply:
German prepositions are a mess ;) But since it here as "der Seite von" in brackets, I think NEXT TO might fit.

Reply:
I will map bei to AT then. However, 715 neben, daneben is already mapped to NEXT TO, and I think this would be the more prototypical expression.

Reply:
I would leave both _Bei_ and _An_ unmapped. Since we do not have any other lists to base our mappings on and the information in the brackets is very specific. But maybe @LinguList can help here?